stored_queries/GO/GO_get_hierarchicalParents.yaml

```diff
@@ -22,21 +22,30 @@ params:
         default: 0
         description: Result offset for pagination
         maximum: 100000
-query_prefix: WITH GO_test_term
+    ts:
+      type: integer
+      title: Versioning timestamp
+query_prefix: WITH GO_terms
 query: |
-  LET term_id = CONCAT("GO_test_term/", @key)
-  LET results1 = (
-    FOR v_relationship, e_relationship IN 1..1 OUTBOUND term_id GO_test_edges_relationship
-      FILTER e_relationship.relationship_type == 'part_of'
-        OR e_relationship.relationship_type == 'has_part'
-        OR e_relationship.relationship_type == 'occurs_in'
-      RETURN {term: v_relationship, edge: e_relationship}
-  )
-  LET results2 = (
-    FOR v_isa, e_isa IN 1..1 OUTBOUND term_id GO_test_edges_isa
-      RETURN {term: v_isa, edge: e_isa}
-  )
-  FOR x IN UNION(results1, results2)
-    SORT x._key ASC
-    LIMIT @offset, @limit
-    RETURN DISTINCT x
+  FOR t in GO_terms
+    FILTER t.id == @id
+    FILTER t.created <= @ts AND t.expired >= @ts
+    limit 1
+    LET results1 = (
+      FOR v_relationship, e_relationship IN 1..1 OUTBOUND t GO_edges_relationship
+        FILTER e_relationship.relationship_type == 'part_of'
+          OR e_relationship.relationship_type == 'has_part'
+          OR e_relationship.relationship_type == 'occurs_in'
+        FILTER e_relationship.created <= @ts AND e_relationship.expired >= @ts
+        RETURN {term: v_relationship, edge: e_relationship}
+    )
+    LET results2 = (
+      FOR v_isa, e_isa IN 1..1 OUTBOUND t GO_edges
+        FILTER e_isa.type == "is_a"
+        FILTER e_isa.created <= @ts AND e_isa.expired >= @ts
+        RETURN {term: v_isa, edge: e_isa}
+    )
+    FOR x IN UNION(results1, results2)
```

Comment:
Similar comments above apply here:
- Merge results1 and reuslts2 into one traversal on `GO_edges` with an extra `OR` clause for the `is_a` type.
- Replace FILTERs on created and expired for edges with a PRUNE 
- Replace `GO_edges_relationship` with `GO_edges` and replace `.relationship_type` with `.type`
- Remove the union since results1 and results2 are now merged